config/value.go

```diff
@@ -201,12 +203,22 @@ func (cv Value) TryAsBool() (bool, bool) {
 
 // TryAsFloat attempts to return the configuration value as a float
 func (cv Value) TryAsFloat() (float64, bool) {
-	f := float64(0)
 	v := cv.Value()
-	if val, err := convertValue(v, reflect.TypeOf(f)); v != nil && err == nil {
+	if val, err := convertValue(v, reflect.TypeOf(_float64Zero)); v != nil && err == nil {
 		return val.(float64), true
 	}
-	return f, false
+	switch val := v.(type) {
+	case int:
+		return float64(val), true
+	case int32:
+		return float64(val), true
+	case int64:
+		return float64(val), true
+	case float32:
```

Comment:
That doesn't seem to work in this case Glib. "cannot convert val (type interface {}) to type int: need type assertion"